src/block/WaterCauldron.php

```diff
@@ -128,9 +129,10 @@ public function onInteract(Item $item, int $face, Vector3 $clickVector, ?Player
 			}else{
 				$this->mix($item, VanillaItems::GLASS_BOTTLE(), $returnedItems);
 			}
-		}elseif($item instanceof Armor){
+		}elseif($item instanceof Armor || $item instanceof HorseArmor){
 			if($this->customWaterColor !== null){
 				if(match($item->getTypeId()){ //TODO: a DyeableArmor class would probably be a better idea, since not all types of armor are dyeable
+					ItemTypeIds::LEATHER_HORSE_ARMOR,
```

Comment:
Alphabetical order would be preferable

Reply:
Are you sure? There is nothing in the alphabetical order I would have to change the order completely and then it says irrelevant change

Reply:
`					ItemTypeIds::LEATHER_HORSE_ARMOR,
					ItemTypeIds::LEATHER_CAP,
					ItemTypeIds::LEATHER_TUNIC,
					ItemTypeIds::LEATHER_PANTS,
					ItemTypeIds::LEATHER_BOOTS => true,`